miniupnpd/minissdp.c

```diff
@@ -65,33 +65,28 @@ AddMulticastMembership(int s, struct lan_addr_s * lan_addr)
 	struct ip_mreqn imr;	/* Ip multicast membership */
 #endif	/* HAVE_IP_MREQN */
 
-    /* setting up imr structure */
-    imr.imr_multiaddr.s_addr = inet_addr(SSDP_MCAST_ADDR);
-    /*imr.imr_interface.s_addr = htonl(INADDR_ANY);*/
+	/* setting up imr structure */
+	imr.imr_multiaddr.s_addr = inet_addr(SSDP_MCAST_ADDR);
```

Comment:
that's not really great to include indentation fixes in the same commit than a functional change...
but thank you anyway ;)

Reply:
sure, I've tried couple of times to make clean patch w/o ident fixes, but incomplete changes of similar codeblocks didn't pass my builtin sense of beauty filter :)
let me split it?

Reply:
please wait a minute...

Reply:
just rebase on the new master ;)

Reply:
okie, thanks